09-ncvs-vignette.Rmd

```diff
@@ -205,7 +207,7 @@ inc_ind %>%
         AAST_Other)
 ```
 
-In the code above, some checking of derived variables is done. This is a good point to pause to look at the output of crosswalks between an original variable and a derived to check the logic was programmed correctly and everything ends up in the bucket you expect. 
+In the code above, some checking of derived variables is done. This is a good point to pause to look at the output of crosswalks between an original variable and a derived to check the logic was programmed correctly and everything ends up in the bucket you expect.  
```

Comment:
Move this above the code where you do this.  There is a LOT of code to skim through without knowing what it is that you are doing/why.